utils/python/CIME/build.py

```diff
@@ -80,7 +79,7 @@ def build_model(case, build_threaded, exeroot, clm_config_opts, incroot, complis
     for model, comp, nthrds, ninst, config_dir in complist:
         if comp == "aquap":
             logger.debug("Now build aquap ocn component")
-            _build_model_thread(config_dir, caseroot, bldroot, libroot, incroot, file_build, 
+            _build_model_thread(config_dir, comp, caseroot, bldroot, libroot, incroot, file_build,
```

Comment:
Were aquaplanet builds simply failing before this change? I guess it's quite possible that no one has been testing them recently.